manifests/candlepin.pp

```diff
@@ -121,31 +118,15 @@
       key_file  => $client_key,
       cert_file => $client_cert,
     } ~>
-    file { $amqp_store_dir:
```

Comment:
I think we should ensure it's absent to clean up after ourselves, but I don't recall if we can ensure directories are absent in Puppet.

Reply:
I wondered about that. Should the puppet module clean this up or installer post install task clean this up?

Reply:
Normally I'd say Puppet but if Puppet can't clean up directories with the `file` type then I'm ok with a post install task rather than an exec here.

Reply:
From my googling you can do a regular file set to absent with recursive set to true. This just doesn't feel as clean. That weird line where sometimes we cleanup in the puppet module and sometimes we use the puppet module to simply represent the state and the installer does cleanup. I thought we were universally doing the latter. Its hard to know which way to go.